product-accelerators/apim/components/org.wso2.healthcare.apim.backendauth/readme.md

```diff
@@ -0,0 +1,64 @@
+# Backend Authentication for WSO2 API Manager
+
+This module provides a streamlined solution for integrating an external authentication server to securely obtain and 
+manage access tokens for backend service requests. By default, WSO2 API Manager's gateway forward the end-user's access token 
+from the invoking client to the backend API. 
+<br>This module replaces that default behavior by fetching a new access token 
+from an external OAuth 2.0-compliant authentication server and replacing the `Authorization` header with the 
+retrieved token.
+
+## Configuration Model
+
+```toml
+[healthcare.backend.auth]
```

Comment:
Readme should have a table to describe the config

Reply:
added in
2baf7e5